shell/platform/embedder/flutter_engine_exports.lst

```diff
@@ -8,7 +8,10 @@
 
 {
   global:
-    *;
+    FlutterDesktop*;
+    FlutterEngine*;
+    FlutterPlatformMessage*;
+    *Dart*;
```

Comment:
How did you get this list?

Also you need to update my comment in line 5 to reflect this change.

Reply:
> How did you get this list?

I added it after seeing the error message during build & running.